docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +0,0 @@
-# Description
```

Comment:
Why you try to delete my fiel :P

Reply:
I think something went wrong and instead of the contact html being just uploaded more files were added